docs/resources/generated-error-pages/error-codes-inventory.rst.inc

```diff
@@ -492,13 +492,163 @@ PACKAGE_UPLOAD_REJECTED
+
+    **Resolution**: Restart the party update procedure by reading this party details again to obtain its most recent state and in particular its most recent resource version.
+
+
+
+
+.. _error_code_INTERNAL_PARTY_RECORD_ALREADY_EXISTS:
+
+INTERNAL_PARTY_RECORD_ALREADY_EXISTS
+---------------------------------------------------------------------------------------------------------------------------------------
+    
+    **Explanation**: Each on-ledger party known to this participant node can a have a participant local metadata assigned to it. The participant local information about a party referred to by this request was found while it should have been not found.
+
+    **Category**: SystemInternalAssumptionViolated
+
+    **Conveyance**: This error is logged with log-level ERROR on the server side. This error is exposed on the API with grpc-status INTERNAL without any details due to security reasons
+
+    **Resolution**: This error can indicate a problem with the server's storage or implementation.
+
+
+
+
+.. _error_code_INTERNAL_PARTY_RECORD_NOT_FOUND:
+
+INTERNAL_PARTY_RECORD_NOT_FOUND
+---------------------------------------------------------------------------------------------------------------------------------------
+    
+    **Explanation**: Each on-ledger party known to this participant node can a have a participant local metadata assigned to it. The participant local information about a party referred to by this request was not found while it should have been found.
+
+    **Category**: SystemInternalAssumptionViolated
+
+    **Conveyance**: This error is logged with log-level ERROR on the server side. This error is exposed on the API with grpc-status INTERNAL without any details due to security reasons
+
+    **Resolution**: This error can indicate a problem with the server's storage or implementation.
+
+
+
+
+.. _error_code_INVALID_PARTY_DETAILS_UPDATE_REQUEST:
+
+INVALID_PARTY_DETAILS_UPDATE_REQUEST
+---------------------------------------------------------------------------------------------------------------------------------------
+    
+    **Explanation**: There was an attempt to update a party using an invalid update request.
+
+    **Category**: InvalidIndependentOfSystemState
+
+    **Conveyance**: This error is logged with log-level INFO on the server side. This error is exposed on the API with grpc-status INVALID_ARGUMENT including a detailed error message
+
+    **Resolution**: Inspect the error details for specific information on what made the request invalid. Retry with an adjusted update request.
+
+
+
+
+.. _error_code_MAX_PARTY_DETAILS_ANNOTATIONS_SIZE_EXCEEDED:
+
+MAX_PARTY_DETAILS_ANNOTATIONS_SIZE_EXCEEDED
+---------------------------------------------------------------------------------------------------------------------------------------
+    
+    **Explanation**: A party can have only a limited amount of annotations. There was an attempt to allocate or update a party with too many annotations.
```

Comment:
```suggestion
    **Explanation**: A party can have only a limited number of annotations. There was an attempt to allocate or update a party with too many annotations.
```

Reply:
The limit works by checking the total number of bytes the annotations occupy, so someone can have one big annotations or a lot of small ones as long as the total number of bytes to store them is below a threshold.

I'll be rephrasing it to:
> A party can have at most 256kb worth of annotations in total measured in number of bytes in UTF-8 encoding.
There was an attempt to allocate or update a party such that this limit would have been exceeded.

Wdyt?

Reply:
"256kb worth" sounds very casual to my ear (as a non-native speaker I am wrong about those things a lot of times though). How about:
> The total size of all party annotations cannot exceed 256kb measured in number of bytes in UTF-8 encoding.
There was an attempt to allocate or update a party such that this limit would have been exceeded.